Felienne/hedy
# [UI IMPROVEMENT] Implements ace editor on programs page

**Description**
In this PR we implement the ace editor on the programs and clean-up the scripts there as well. For some reason the client side messages where not included on this page, creating a possible bug where no feedback would be returned to the user on an unexpected error. But most importantly, we now support syntax highlighting on the programs page.


**Fixes**
This PR fixes #2495.

**How to test**
Navigate to the `/programs` page and notice the new ace editors for each stored program. We limit the program shown to 4 lines, the rest is already discarded in the back-end.
